doc/source/conf.py

```diff
@@ -589,6 +590,25 @@ def get_version_match(semver):
 copybutton_prompt_text = r'>>> ?|\.\.\. '
 copybutton_prompt_is_regexp = True
 
+# sphinx-tags options ---------------------------------------------------------
+# See https://sphinx-tags.readthedocs.io/en/latest/index.html
+
+tags_badge_colors = {
+    "Load": "primary",
+    "Filter": "secondary",
+    "Plot": "dark",
+    "Widgets": "success",
+    "Lights": "primary",
```

Comment:
Should we also make these lowercase?
```suggestion
    "load": "primary",
    "filter": "secondary",
    "plot": "dark",
    "widgets": "success",
    "lights": "primary",
```